frontend/src/components/GameComponent/gameTop/Game.js

```diff
@@ -2,7 +2,7 @@ import '../game.css';
 import img from '../../asseets/rdr2.jpg'
 import {LoremIpsum} from 'react-lorem-ipsum';
 
-function Game() {
+function Game(props) {
```

Comment:
instead of (props) try ({name, lastName, img}) this way you don't have to use `props.__name__` everywhere in react component.